Exercise_nonsequential/array_job_nonsequence.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This script runs the array job spawned using the main.sh script on a
```

Comment:
@manasvita best practice is to put the `#SBATCH` lines right after `#!/bin/bash` . I remember there was a ticket where a user's job didn't pick up all of the `#SBATCH` directives because there were either spaces in between or some code above `#SBATCH` . So just to be sure I would change it.

Reply:
@marianalevineu , thanks for your comments. I think I saw that ticket too but @gshomo and I had verified that putting comments after #!/bin/bash has no effect on #SBATH directives when we were working on the recursive job array script. So, this is all good.

Reply:
@marianalevineu, @manasvita, the [sbatch documentation](https://slurm.schedmd.com/sbatch.html) mentions: "sbatch will stop processing further #SBATCH directives once the first non-comment non-whitespace line has been reached in the script."  This means it's fine to have comments interspersed with `#SBATCH` directives after `#!/bin/bash` but before any actual commands.